test/test-creation.js

```diff
@@ -1405,8 +1217,12 @@ describe('JHipster client-2 generator', function () {
         it('creates expected files for default configuration for client-2 generator', function () {
             assert.noFile(expectedFiles.server);
             assert.noFile(expectedFiles.maven);
-            assert.file(expectedFiles.client_2);
-            assert.file(['gulpfile.js']);
+            assert.file(getFilesForOptions(require('../generators/client-2/files').files, {
```

Comment:
@jhipster/developers this is the best part of this change. we need not maintain list of files to check in tests anymore it can be obtained dynamically for any config required 😄